README.md

```diff
@@ -0,0 +1,27 @@
+# Music Library
+An app for managing a music library.
+
+## Motivation
+A backend project with MCRCodes to learn to develop an API which can perform CRUD operations on a database.
+
+## Technologies
+* Javascript
+* MySQL
+* Express
+
+Deveopment and Testing:
+ * Docker
+ * Node, Nodemon
+ * MySQL Workbench
+ * Postman
+ * Supertest
+ * Mocha
+ * Chai
+
+
+## Build Status
```

Comment:
What should the `.env` file contain? How is the project setup? What scripts do I need to run?

These are all questions a newcomer to this repo would need to know the answers to, so they should be included in the README